experimental/generator-dotnet-yeoman/generators/app/templates/functions/Startup.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I assume the practice for template/generator/yadda yadda is to not include copyright headers?